website/static/css/pages/home-page.css

```diff
@@ -39,6 +36,11 @@ h3, h4 {
     color: #FFF;
 }
 
+.banner {
```

Comment:
Since this is for a button, perhaps we could call it `btn-banner` to match with the naming scheme.